prometheus_client/exposition.py

```diff
@@ -37,7 +37,7 @@
     'write_to_textfile',
 )
 
-CONTENT_TYPE_LATEST = 'text/plain; version=0.0.4; charset=utf-8'
+CONTENT_TYPE_PLAIN = 'text/plain; version=0.0.4; charset=utf-8'
```

Comment:
text/plain v 0.0.4 is definitely not the latest, so I thought it best to rename this

Reply:
Though I agree with you, this will be a breaking change as other programs use `CONTENT_TYPE_LATEST`. Perhaps we should update this to version=1.0.0 instead?

Reply:
Since this is needed for compatibility, but LATEST and PLAIN are both really different versions of PLAIN that is confusing for external users to use. I'd propose a couple options:
1. Keep `CONTENT_TYPE_PLAIN` internal, so rename it to `_CONTENT_TYPE_PLAIN`.
2. Support content types for each version, so have `CONTENT_TYPE_PLAIN_0_4_0`, `CONTENT_TYPE_PLAIN_1_0_0`, and then alias `CONTENT_TYPE_LATEST` to `CONTENT_TYPE_PLAIN_1_0_0`.

Reply:
I like option 2